pkg/controllers/managementuserlegacy/cis/clusterScanHandler.go

```diff
@@ -458,6 +458,7 @@ func (csh *cisScanHandler) ensureCleanup(cs *v3.ClusterScan) error {
 	podName := runnerPodPrefix + cs.Name
 	if e := csh.podClient.Delete(podName, &metav1.DeleteOptions{}); e != nil && !kerrors.IsNotFound(e) {
 		err = multierror.Append(err, fmt.Errorf("cis: ensureCleanup: error deleting pod %v: %v", podName, e))
+		_ = err
```

Comment:
I think the issue is the if statement puts this into a new context which the linter doesn't like, so if the if check is dropped the linter won't error

Reply:
I brought the `e` and `err` in line with the other statements and it's fine now, unless I broke it in some other way.